controllers/metal3machine_controller_integration_test.go

```diff
@@ -416,6 +416,42 @@ var _ = Describe("Reconcile metal3machine", func() {
 			},
 		),
 		//Given: Machine has Bootstrap data available while BMMachine has no Host Annotation
+		// BMH is in available state
+		//Expected: Requeue Expected
+		//			BMHost.Spec.Image = BMmachine.Spec.Image,
+		// 			BMHost.Spec.UserData is populated
+		// 			Expect BMHost.Spec.ConsumerRef.Name = BMmachine.Name
+		Entry("Should set BMH Spec in correct available state and requeue when all objects are available but no annotation",
```

Comment:
Can you please make this entry name unique too please ? It is the same name as the following entry.